installers/install.sh

```diff
@@ -168,6 +168,11 @@ if $FORCEOVERWRITE && ( ! $NOPROMPT ); then
   exit 1
 fi
 
+echo "\
+\033[2m╔═══════════════════════╗
```

Comment:
`2m` or `32m` like in the box below?

Reply:
`2m`. The box below is a success state so its outline is green.